tests/test_models.py

```diff
@@ -57,4 +57,4 @@ def runTest(self):
 
 
 if __name__ == '__main__':
-    unittest.main()
+    unittest.main()
```

Comment:
Was there meant to be a test added in this file?

Reply:
Yes, still need to do it. Actually I never wrote unit tests before. Will do it.

Reply:
@FooSoftm Hey, I added unit test, let me know if that is correct cause I am a newbie writing unit tests.

Reply:
Added unit tests. Let me know if it is correct cause I am a newbie for writing unit tests. Thanks

Reply:
Tests themselves look fine I think. 👍🏻